specs/001-docker-dev-stack/quickstart.md

```
@@ -251,15 +263,15 @@ docker compose logs directus
-| Service | URL | Credentials | Purpose |
-|---------|-----|-------------|---------|
-| **Directus** | http://localhost:8055/admin | Email/Password from `.env` | CMS admin panel |
-| **Ghost** | http://localhost:2368/ghost/ | Setup on first visit | Blog editor |
-| **Astro** | http://localhost:3000/ | None (public site) | Frontend preview |
-| **Listmonk** | http://localhost:9000/admin | Setup on first visit | Email management |
-| **Mixpost** | http://localhost:8080/ | Setup on first visit | Social scheduling |
-| **PostgreSQL** | localhost:5432 | `postgres` / `.env password` | Database (via client) |
-| **Redis** | localhost:6379 | None | Cache (via client) |
+| Service | URL | Credentials | Purpose | Status |
+|---------|-----|-------------|---------|--------|
+| **Directus** | http://localhost:8055/admin | Email/Password from `.env` | CMS admin panel | ✅ Active |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)